lib/BaseController.js

```diff
@@ -19,11 +19,25 @@ const clone = require('clone');
 const merge = require('deepmerge');
 const fs = require('fs-extra');
 const hash = require('object-hash');
-
+const BOOL_EXPR = /^(true|false)$/;
 
 module.exports = class BaseController {
   constructor(params) {
 
+    let reconcileByDefault = 'true';
+
+    if (objectPath.has(params, 'reconcileByDefault')) {
```

Comment:
with the assumption that this value comes from an environment variable, checking for the presence of the key might create a lot of friction and re-checking everywhere to omit the key. This is probably better of as a loose `== null` check